apps/accounts/models/hosting.py

```diff
@@ -263,6 +300,30 @@ class Hostingprovider(models.Model):
     def __str__(self):
         return self.name
 
+    @property
+    def users(self) -> models.QuerySet["User"]:
+        """
+        Returns a QuerySet of Users who have permissions for a given Hostingprovider
+        """
+        return get_users_with_perms(
+            self, only_with_perms_in=(manage_provider.codename,)
+        )
+
+    @property
+    def users_explicit_perms(self) -> models.QuerySet["User"]:
+        """
+        Returns a QuerySet of all Users that have *explicit* permissions to manage this Hostingprovider,
+        not taking into consideration:
+            - group membership
+            - superuser status
+        """
+        return get_users_with_perms(
```

Comment:
Would you please add a line in the code commentsd about why we have this function?

It's not really needed in the docstring, but it's not obvious that without it, the admin screens become a horror show of massive listings of providers a user has access to, and knowing this would be helpful for understanding why we have two functions that seem to do similar things.